### ISSUE TITLE
Going to Storage tab in host details page causes Error page

### ISSUE THREAD

**Issue by tfiduccia** [rancher/rancher#5982]
**Rancher Version:** v1.2.0-pre3

**Steps to Reproduce:**
1.  Go to Host details page
2.  Click on Storage tab

**Results:** Error e.forEach is not a function
![image](https://cloud.githubusercontent.com/assets/11514927/18567417/ddd60942-7b4c-11e6-8f63-aa583875b3fc.png)

**Expected:** No error

**Comment by argent-smith** [comment]
Same on 1.1.4 stable (just upgraded from 1.1.3 stable)

**Comment by kevin-wang-cy** [comment]
Rancher	v1.1.4
Cattle	v0.165.8.4
User Interface	v1.1.14
Rancher Compose	v0.8.6

I got same error just now

**Comment by tfiduccia** [comment]
Version - 1.2.0-pre4-rc2
Not able to duplicate

**Comment by vincenzobluesparklabs** [comment]
Got the same error with
Rancher 1.1.4
EFS Convoy

**Comment by Bekt** [comment]
Seeing this as well.

```
Rancher	v1.1.4
User Interface	 v1.1.14
OS: CoreOS 1122.2.0
Docker: 1.10.3
```


```
Application Error TypeError: e.forEach is not a function
    at Array.addObjects (.../assets/vendor-ebfa7dda04b9f2b654291bf27e70bf22.js:14:20536)
...
```

**Comment by Latnok** [comment]
same error

---
| Useful | Info |
| :-- | :-- |
|Versions|Rancher `v1.1.4` Cattle: `v0.165.8.4` UI: `v1.1.14` |
|Access|`Disabled`|
|Orchestration|`Cattle`|
|Route|`host.containers`|

**Comment by harshal-shah** [comment]
Facing same issue with following: 
![deleteme](https://cloud.githubusercontent.com/assets/16479600/19657644/ac8096e0-9a42-11e6-8cd6-c82ea4117613.jpg)

**Comment by deniseschannon** [comment]
@harshal-shah Yes, it is fixed in v1.2.0-pre4-rc2+ and will be available in the next release.
